terraform.tf

```
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "> 3.71, <= 3.99.0"
+      version = ">= 4.4.0"
```

To avoid issues with azurerm v5 please can you change this to `~> 4.4`?

```suggestion
      version = "~> 4.4"
```